src/main/java/dukepro/DukePro.java

```diff
@@ -0,0 +1,165 @@
+package dukepro;
+
+import dukepro.exceptions.DukeException;
+import dukepro.handlers.Interact;
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
```

Comment:
A very minor nitpick but I would love to see some separation between imports from different packages for readability